packages/google_maps_flutter/google_maps_flutter_ios/ios/google_maps_flutter_ios.podspec

```diff
@@ -22,8 +22,15 @@ Downloaded by pub (not CocoaPods).
   # has been confirmed to be compatible via an example in examples/. See discussion
   # in https://github.com/flutter/flutter/issues/86820 for why this is so broad.
   s.dependency 'GoogleMaps', '< 9.0'
+  # Once the plugin requires iOS version 13+, this should be updated.
+  s.dependency 'Google-Maps-iOS-Utils', '4.1.0'
   s.static_framework = true
   s.platform = :ios, '12.0'
+  # Enabling Swift support for Google-Maps-iOS-Utils
+  s.swift_version = '5.0'
```

Comment:
why is swift required to use this library? this doesn't look like a swift only library.

Reply:
Swift is required by the utils library (v4.1.0): https://github.com/googlemaps/google-maps-ios-utils/tree/v4.1.0:

Without following lines:
```
  s.swift_version = '5.0'
  s.xcconfig = {
    'LD_RUNPATH_SEARCH_PATHS' => '$(inherited) /usr/lib/swift',
  }
```

Following exception is thrown:
```
Exception Type:  EXC_CRASH (SIGABRT)
Exception Codes: 0x0000000000000000, 0x0000000000000000
Termination Reason: DYLD 1 Library missing
Library not loaded: @rpath/libswiftCore.dylib
```

Seems like this works without the swift_version line `s.swift_version = '5.0'`
But i think it should be kept in as well to match the swift version on utils lib:
(https://github.com/googlemaps/google-maps-ios-utils/blob/v4.1.0/Google-Maps-iOS-Utils.podspec#L22)
@stuartmorgan any comments on this?

Reply:
You may wanna put `5.3` rather than `5.0` from the link you posted: 

> Note: This feature is only available with Swift 5.3 (Xcode 12) or later.

Reply:
It seems to be related to the Swift Package Manager